networking/ovn_kubernetes_network_provider/converting-to-dual-stack.adoc

```diff
@@ -11,7 +11,7 @@ After converting to dual-stack, all newly created pods are dual-stack enabled.
 
 [NOTE]
 ====
-A dual-stack network is supported on clusters provisioned on only installer-provisioned bare metal infrastructure.
+A dual-stack network is supported on clusters provisioned on only installer-provisioned bare metal infrastructure as well as Single Node OpenShift clusters.
```

Comment:
Can u check how we call the bm sno in the docs and specify it here in the same manner?

Reply:
All I have found is https://docs.openshift.com/container-platform/4.9/installing/installing_sno/install-sno-installing-sno.html which mentions it as "OpenShift on a single node"